RasterPropMonitor/Core/MonitorPage.cs

```diff
@@ -576,11 +577,12 @@ public void RenderBackground(RenderTexture screen)
                     }
                     break;
                 case BackgroundType.Texture:
+                    //call clear before redraw of textures
                     GL.Clear(true, true, ourMonitor.emptyColorValue);
                     Graphics.DrawTexture(new Rect(0, 0, screen.width, screen.height), backgroundTexture);
                     break;
                 case BackgroundType.Handler:
-
+                    //No clear here as it would interfere with the handlers(Causing effects such as VesselViewer to blink)
```

Comment:
Ahh yes, that's the only case which I hadn't tested, if it ran out of power. Other than that, Plugins such as VesselViewer no longer blinked, solid texture screens were drawn correctly and blank "text" screens were as well.